### DISCUSSION TITLE
[github-actions] bot locked conversation against mentions

### DISCUSSION THREAD

**Discussion by TonyGravagno** [community/community#29083] [Actions]
After "(https://github.com/apps/github-actions) bot locked and limited conversation to collaborators", a canonical thread no longer receives `#00000` mention notifications. I believe mentions should be indicated in threads locked by the actions bot.

![image](https://user-images.githubusercontent.com/499473/184029269-33d68e41-a583-47e0-ad3b-a5fb63dca11e.png)

Assume a PR is merged. The issue is closed, as is the PR record. Then they get locked. But some months later the PR is suspected of introducing a new defect. A new Issue uses `#00000` markup to reference the PR and the Issue it intended to fix. A normally Closed canonical Issue gets annotated with a mention to the new Issue. But because the threads are locked, there is no mention visible to non-contributors.

Do contributors to the original/canonical locked Issue get notified of mentions?  
Do they see a mention in the Issue even if no one else does?

I believe all mentions should be public. Anyone who is researching a defect might find a related Issue/thread that happens to be locked by the bot. There might be other open Issue threads referring to that canonical Issue - but right now they aren't mentioned. That can lead to more than one new report, and unnecessary duplication.

On the other side of this, since we can reference an issue even after it's been locked, on triage of a new report one of the collaborators might agree that there is a relationship and manually add their own reference to the original. This is the way it works now. I just think it's a bit chaotic when it doesn't need to be. If someone cares enough to link topics that seem to be related, let's make as much use of that information as possible.

**Comment by fedeci** [comment]
> Do contributors to the original/canonical locked Issue get notified of mentions? Do they see a mention in the Issue even if no one else does?

The answer is no to both the questions. Even collaborators of the repo cannot see the mention.

**Comment by zong-zhe** [comment]
Hi @fedeci :smile: , Can this automatic process be turned off? I want to know how to make github action bot not lock after PR merged.
